### PR TITLE
Close #409: Fix FAQ typo and add more FAQ

### LINKED ISSUE #409: Typo on FAQ section
Discord member found typo

image

### PR DESCRIPTION
**Pull request by aminick**

### CODE REVIEW

**Comment by aminick** [comment]
Frequently_Asked_Questions___Flexpool_io

File: src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ title: Does Flexpool charge a fee for every transaction?
 coin: xch
 ---
 
-**No**. We don't charge any additional transaction fees from our side. The only fee you will pay is the network transaction fee, that is static regardless of the transaction value.
+**No**. There are no transaction fees paid by the farmer for XCH transactions.
```
Comment (AlexSSD7) [inline src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md:6]: Right now yes, there are no fees. But in future, highly likely fees will appear as more people will start using Chia.

**Review by AlexSSD7 — CHANGES_REQUESTED** [review]

File: src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md (hunk 1 above)
Comment (aminick) [reply src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md:6]: Do you want me just remove this FAQ completely? 
Or I can change the working to "Right now, there are no .......", but seems subtle.

Comment (AlexSSD7) [reply src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md:6]: Both options will work.

Comment (aminick) [reply src/docs/ar/faq/questions-about-flexpool/xch-does-flexpool-have-transaction-fees.md:6]: Removed

**Review by AlexSSD7 — APPROVED** [review]
LGTM

**Pull request merged**